samples/bot-ai-meeting-helper/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+    {
+      "name": "officedev-microsoft-teams-samples-bot-ai-meeting-helper-nodejs",
```

Comment:
This is not mentioned in readme Fragment section.

Reply:
Done